Jenkinsfile.deploy

```diff
@@ -231,6 +229,22 @@ pipeline {
     }
 }
 
+void buildOptaweb(MavenCommand optawebCommand){
+    MavenCommand buildCommand = optawebCommand
+    if (!isRelease()) {
```

Comment:
just thinking that for clarity, as you start to have a param for that job, the decision should be taken by [the main release pipeline](https://github.com/kiegroup/kogito-pipelines/blob/master/Jenkinsfile.release#L95) and not here. 
So the release pipeline would send `true` to `SKIP_INTEGRATION_TESTS` parameter.

So we don't have that particular case here in that file and we keep the visibility in the job `optaplanner-deploy` parameters directly.
That could be confusing to have set `SKIP_INTEGRATION_TESTS` to `false` but they are not executed because `env.RELEASE` is true and you don't get why.
wdyt ?

Reply:
Fixed in 359500519c755a45724df3f52eb38e21844abfdc
Related PR
https://github.com/kiegroup/kogito-pipelines/pull/135